docs/src/components/componentList/drawerNavItemComponent/DrawerNavItemWithIcons.tsx

```diff
@@ -31,7 +30,18 @@ export const DrawerNavItemWithIcons = (): JSX.Element => {
                 </Drawer>
             </Box>
             <CodeBlock code={codeSnippet} language="jsx" dataLine="3-5" />
-            <CopyToClipboard title={'Copy All'} copyText={codeSnippet} />
+            <Box sx={{ display: 'flex', justifyContent: 'flex-end' }}>
+                <CopyToClipboard title={'Copy All'} copyText={codeSnippet} />
+                <Button
+                    sx={{ ml: 2 }}
+                    variant="outlined"
+                    target="_blank"
+                    href="https://codesandbox.io/s/drawernavitemwithicons-7k0oiy?file=/src/DrawerNavItemWithIconsExample.tsx"
+                    startIcon={<CodeIcon />}
+                >
+                    SandBox
+                </Button>
```

Comment:
Do we want to componentize this button as well if we are planning to have codesandbox links?